Shared/Extensions/UI Extensions/UICollectionView+Extensions.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  UICollectionView+Extensions.swift
+//  UICollectionView+Extensions
+//
+//  Created by Martin Young on 8/24/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+extension UICollectionView {
```

Comment:
I grabbed these from the old CollectionViewManager and made them async. We're dumbing down the collection view manager so it doesn't need these any more.